data/campaigns/atl02.wmf/scripting/texts.lua

```diff
@@ -0,0 +1,743 @@
+   },
+}
+
+heretics_again = {
+   {
+      title = _("Heretics!"),
+      body = sidolus(
+         -- TRANSLATORS: Sidolus
+         _([[May Satul warm us all. Our princess has been confused in her prisonship by this barbarian. But I will never worship any barbarian god. I can’t imagine how we could change her mind though.]]))
+   },
+   {
+      title = _("Attracting Satuls Help"),
+      body = loftomor(
+         -- TRANSLATORS: Loftomor
+         _([[Sidolus, I agree we should not bow in front of any other god then Satul. But we might need the gods help in this case. So I suggest to build a temple and worship satul with exquisite sacrifices. By this we could inaugurate some priests, who might beg Satul to show his powers to the barbarians and our princess.]]))
+   },
+   {
+      title = _("Build Temple"),
+      body = sidolus(
+         -- TRANSLATORS: Sidolus
+         _([[May Satul warm you Loftomor. This is a very good idea indeed. So I task you to build a temple worth worshipping our god. Until we are ready to beg for Satuls power we shopuld keep an eye on our princess and this barbarian. As we should stop our traders deliveries to not waste any wares to this barbarian anymore, I want you to ensure we have a scout out there to gather information.]]))
+         .. new_objectives(obj_build_temple),
+   },
+}
+
+temple_completed = {
+   {
+      title = _("Worship Satul"),
+      body = loftomor(
+         -- TRANSLATORS: Loftomor
+         _([[May Satul warm you, Sidolus! My builders just completed the temple for Satul. Now we can start worshipping him and consecrating our priests.]]))
+   },
+   {
+      title = _("Build Temple"),
+      body = sidolus(
+         -- TRANSLATORS: Sidolus
+         _([[May Satul warm you, Loftomor. Thank you for building such a great temple. Now we need to bring food and our most precious wares to the temple to make for satisfying sacrifices. And don’t forget some golden tabards for the priests.]]))
+         .. new_objectives(obj_worship),
+   },
+}
+
+priests_consecrated = {
+   {
+      title = _("May Satul Warm Us"),
+      body = sidolus(
+         -- TRANSLATORS: Sidolus
+         _([[Our priests have been consecrated, and we sacrificed our finest goods for Satuls pleasure. May he warm us and show his might to our princess and these barbaric tribe.]]))
+   },
+}
+
+princess_back = {
+   {
+      title = _("Jundlina Arrives"),
+      body = jundlina(_("Jundlina"),
+         -- TRANSLATORS: Jundlina
+         _([[May Satul warm you, Sidolus. And may he forgive me my heretic behaviour. The sign of his might and the fear in the barbarians’ faces suddenly made me recognise the truth again. I was wrong about peace and coexistence, as it would have meant knightship forever. And this is not what Atlanteans are made for.]]))
```

Comment:
```suggestion
         _([[May Satul warm you, Sidolus. And may he forgive me my heretic behaviour. The sign of his might and the fear in the Barbarians’ faces suddenly made me recognise the truth again. I was wrong about peace and coexistence, as it would have meant servitude forever. And this is not what Atlanteans are made for.]]))
```